Sources/Noora/Noora.swift

```diff
@@ -33,12 +33,46 @@ public protocol Noorable {
     ///   - collapseOnSelection: When true, the question is collapsed after the question is entered.
     /// - Returns: The option selected by the user.
     func yesOrNoChoicePrompt(
-        title: String?,
-        question: String,
+        title: TerminalText?,
+        question: TerminalText,
         defaultAnswer: Bool,
-        description: String?,
+        description: TerminalText?,
         collapseOnSelection: Bool
     ) -> Bool
+
+    /// It shows a success alert.
+    /// - Parameters:
+    ///   - message: The success message
+    ///   - next: A list of steps that the person could take after.
+    func success(_ message: TerminalText)
+
+    /// It shows a success alert.
+    /// - Parameters:
+    ///   - message: The success message
+    ///   - next: A list of steps that the person could take after.
+    func success(_ message: TerminalText, next: [TerminalText])
+
+    /// It shows an error alert.
+    /// - Parameters:
+    ///   - message: The error message
+    ///   - next: A list of steps that the person could take after.
+    func error(_ message: TerminalText)
+
+    /// It shows an error alert.
+    /// - Parameters:
+    ///   - message: The error message
+    ///   - next: A list of steps that the person could take after.
+    func error(_ message: TerminalText, next: [TerminalText])
+
+    /// It shows a warning alert.
+    /// - Parameters:
+    ///   - messages: The warning messages.
+    func warning(_ messages: [TerminalText])
+
+    /// It shows a warning alert.
+    /// - Parameters:
+    ///   - messages: The warning messages.
+    func warning(_ messages: [(TerminalText, next: TerminalText?)])
```

Comment:
I'd consider creating a `struct Message: ExpressibleByStringLiteral` to allow the following API:
```swift
Nora().warning(["Simple warning", .message("Complex warning", nextSteps: "Here are some next steps")])
// Instead of how you would have to do this now
Nora().warning([("Simple warning", next: nil), ("Complex warning", next "Here are some next steps")])
```